sr_unix/ydbenv.mpt

```diff
@@ -90,6 +90,7 @@ set	; Set up environment (environment variables, database, etc.) with default st
 	. . for  read tmp1 quit:$zeof
 	. . use "dbcreateerr" for  read tmp2 quit:$zeof
 	. . use io close "dbcreate"
+	zsy "mkdir -p "_env("ydb_dir")_"/r"
```

Comment:
_YDBENV.m already does this, e.g.:

    bhaskar@zomok:~$ rm -rf /tmp/ydb_test
    bhaskar@zomok:~$ export ydb_dir=/tmp/ydb_test
    bhaskar@zomok:~$ source /opt/yottadb/r110/ydb_env_set
    bhaskar@zomok:~$ echo $ydb_dir
    /tmp/ydb_test
    bhaskar@zomok:~$ ls -l $ydb_dir
    total 0
    drwxr-xr-x 2 bhaskar gtc  40 Feb 26 14:21 r
    drwxr-xr-x 5 bhaskar gtc 100 Feb 26 14:21 r1.10_x86_64
    lrwxrwxrwx 1 bhaskar gtc  12 Feb 26 14:21 V6.3-003A_x86_64 -> r1.10_x86_64
    bhaskar@zomok:~$ 

So this addition is not needed.

Reply:
I have a feeling that this is another one of those shell differences between what is ran in docker vs your local host. I know that every time this is ran in a docker container it doesn't have the r directory created in the ydb_dir. I suspect that it was meant to be created by: https://github.com/YottaDB/YottaDB/pull/162/files#diff-196c7fdca0c8742d5982d856c5128348R276

I can put some more debugging statements in GetDefaults to see if there is an attempt to create the directory, all I can relay is what the output I get is :).